benchmark.md

```diff
@@ -11,20 +11,18 @@
 ### OCaml 4.13.1
 
 ```sh
-opam switch 4.13.1
-eval $(opam env)
-dune build --profile=release
-dune exec bin/sdl2/bench.exe -- resource/games/tobu.gb --frames 1500
+opam switch . ocaml-base-compiler.4.13.1
```

Comment:
I get the following error message when I run this command:
```
opam: Invalid switch subcommand "."
Usage: opam switch [OPTION]... [COMMAND] [ARG]...
Try `opam switch --help' or `opam --help' for more information.
```
Any idea on why this is happening? I tried with both opam version `2.0.5` and `2.1.2`.

Reply:
Oops... I meant `opam switch create . ocaml-base-compiler.4.13.1`! But not really!